.github/workflows/slow-tests.yml

```diff
@@ -72,6 +72,8 @@ jobs:
           # transformers requires `tf-keras` when keras 3.0 is installed.
           # See https://github.com/huggingface/transformers/issues/27377 for more details.
           python -m pip install langchain_experimental tf-keras
+          # Installing pyarrow < 18 to prevent lightgbm test pip installation resolution conflicts.
+          python -m pip install "pyarrow<18"
```

Comment:
```suggestion
          # Installing pyarrow < 18 to prevent lightgbm test pip installation resolution conflicts.
          python -m pip install langchain_experimental tf-keras "pyarrow<18"
```